scripts/model_gen_ng/model_gen_ng/jinja_utils.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+import jinja2
```

Comment:
Can you please make a list of all the Python deps we have in `revng`?
I'd like to have in CMake a `python -c 'import $PACKAGE'` to detect a configure time the missing dep.
Also, we need to update the orchestra config.

Reply:
See the `python/CMakeLists.txt` (which uses a `requirements.txt` in the same folder)